Core.PoseEditor/MainWindow.cs

```diff
@@ -1665,7 +1671,22 @@ private void LoadSceneGeneric(XmlNode node, Dictionary<int, ObjectCtrlInfo> dic)
                     continue;
                 }
 
-                if (dic[index] is OCIItem ociItem)
+                retryInfoGet:
+                
+                var item = dic.GetValueSafe(index);
+                
+                if (item == null)
+                {
+                    if (changeDictionary != null && changeDictionary.TryGetValue(index, out var newIndex))
+                    {
+                        index = newIndex;
+                        goto retryInfoGet;
```

Comment:
Wouldn't it be more proper to check the changeDictionary first? Can there be a situation where index exists in dic and in changeDictionary, and if yes then which one is the correct one then?

Reply:
Yeah you're right actually. Well, it can be handled on a case by case now that I look it over again. On import a dictionary is created that lists items imported last. Since PE handles import separately, just gotta only process the imported items and feed it the old keys instead of the new keys with the new ObjectInfos. On serialization, new keys should be used anyway and it becomes part of the "normal" scene.